docs/setup-cluster/slurm/hpc-with-agent.rst

```diff
@@ -72,16 +72,15 @@ Before users can make use of Determined agents, a system administrator must prov
 
 This step may be completed either by the system administrator or the intended user. In a
 cluster-wide shared directory (examples in this section use ``$HOME``), create an ``agent.yaml``
-file. Below is a minimal example using a resource pool named for the user (``$USER``) and
-``singularity`` as the container runtime platform. If configured using variables such as ``$HOME``,
-a single ``agent.yaml`` could be shared by all users.
+file. Below is a minimal example using a resource pool named for the user (``$USER``). ``docker``
```

Comment:
I don't think this will work

I think in slurm environments it is very likely that the launched jobs won't have access to docker so the agent method won't be supported

I think this whole ```hpc-with-agent``` workload is no longer going to be supported, so I think we can remove all these docs

Reply:
would this entail removing the page?: https://docs.determined.ai/latest/setup-cluster/slurm/hpc-with-agent.html

if so, please create a redirect. there is a guide here on using the docs redirect tool https://hpe-aiatscale.atlassian.net/wiki/spaces/DOC/pages/1338310660/How+to+Use+the+Docs+Redirect+Tool

Reply:
let me know how i can help

Reply:
While using redirect to remove a .rst file, what do we update after the colon ':' in [this](https://github.com/determined-ai/determined/blob/fda515d33b44d1d4f1c29ebcce4b59d872d8630f/docs/.redirects/redirects.json#L4) file?
`"docs/setup-cluster/slurm/hpc-with-agent.rst" : "_index.html", `
Does the above code look right?

[Redirect removing file doc](https://hpe-aiatscale.atlassian.net/wiki/spaces/DOC/pages/1338310660/How+to+Use+the+Docs+Redirect+Tool#Removing-a-File)
It mentions about if we have moving a file from one directory location to other, but doesn't mention anything about when we remove the file completely?

Reply:
Remove the page with `git rm`
1. run `git rm docs/setup-cluster/slurm/hpc-with-agent.rst`

Remove the toctree element since the page is gone
2. Edit the `docs/setup-cluster/slurm/_index.rst` page to remove the toctree element `hpc-with-agent`

Create a new redirect by editing an existing redirect
3. modify the `docs/.redirects/redirects.json` file, replacing line 16
    "setup-cluster/deploy-cluster/slurm/hpc-with-agent": "../../slurm/hpc-with-agent.html", 
with
    "setup-cluster/deploy-cluster/slurm/hpc-with-agent": "../../slurm/_index.html",

Should build with no errors after that.

Reply:
@tara-det-ai -

Error occurred while building after making the above suggested change-

> % make -C docs check                               
> git ls-files -z '*.rst' | xargs -0 rstfmt -w 100 --check
> python3 redirects.py check
> check failed; the following previously-published urls seem to have been dropped and should be assigned redirects:
>     setup-cluster/slurm/hpc-with-agent
> make: *** [check] Error 1

The above was solved when I added another line (reference to `setup-cluster/slurm/hpc-with-agent` in `docs/.redirects/redirects.json` file.
`"setup-cluster/slurm/hpc-with-agent": "../slurm/_index.html",`

Let me know if this looks right.